eng/azure-pipelines.yml

```diff
@@ -27,6 +27,7 @@ stages:
       enablePublishBuildArtifacts: true
       enablePublishBuildAssets: true
       enablePublishUsingPipelines: true
+      enableSourceBuild: true
```

Comment:
This would be the right place to specify source-build platform - right below `enableSourceBuild: true` line:
```
      sourceBuildParameters:
        platforms:
        - name: 'AlmaLinux_8_Portable'
          container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:almalinux-8-source-build'
```